src/app/grants/mainGrant.tsx

```diff
@@ -15,11 +14,41 @@ export default function MainGrant({
   return (
     <div className="flex flex-col">
       <GrantsHero />
-      <Eligibility />
-      <StartGrant setPage={setPage} />
-      <Perks />
-      <Cohort setPage={setPage} />
-      <CohortStructure />
+      <BtrustGrant />
+      {/* <Eligibility /> */}
+      <AllGrants setPage={setPage} />
+      <FaqContainer
+        faqs={[
+          {
+            answer:
+              "Btrust Grants provide financial support to developers, educators, and event organizers working to strengthen the Bitcoin open-source ecosystem. We offer grants for developer contributions, education programs, research, and Bitcoin-only events to remove financial barriers and enable impactful contributions.",
+            question: "What are Btrust Grants?",
+          },
+          {
+            answer:
+              "Our grants are open to developers, educators, and organizers from Africa, India, Latin America, MENA, and Southeast Asia. We also consider applicants from other Global South regions on a case-by-case basis, contingent on their Proof of Work.",
+            question: "Who can apply for a Btrust Grant?",
+          },
+          {
+            answer: "We fund:",
+            question: "What kind of projects do you fund?",
+            lists: [
+              "Open-source Bitcoin development and research.",
+              "Developer training, language translation, and Bitcoin-related research.",
+              "Bitcoin-only meetups, bootcamps, and developer-focused conferences.",
+            ],
+          },
+          {
+            answer: "All grants are paid in Bitcoin (BTC)",
```

Comment:
To clarify the "how" and still communicate we give it all in BTC:

```suggestion
            answer: "All grants are paid monthly or one-time (depending on the grant) in Bitcoin (BTC)",
```